backend/api/generic.go

```diff
@@ -77,6 +96,20 @@ func (g *GenericClient) SetTypeMeta(meta corev2.TypeMeta) error {
 	return nil
 }
```

Comment:
Ran into an issue while testing. I believe the `SetTypeMeta` method needs to use `types.ResolveRaw` instead.

Reply:
or, ... the typemap needs to return the proxied resource.

Reply:
I made it work with `ResolveRaw`, should be good to go now :+1: